internal/rootcoord/dml_channels.go

```diff
@@ -23,13 +23,16 @@ import (
 	"sync"
 
 	"github.com/milvus-io/milvus/internal/metrics"
+	"github.com/milvus-io/milvus/internal/util/paramtable"
 
 	"go.uber.org/zap"
 
 	"github.com/milvus-io/milvus/internal/log"
 	"github.com/milvus-io/milvus/internal/mq/msgstream"
 )
 
+const preCreatedTopicPrefix = "pre-created-topic"
```

Comment:
check all usages of namePrefix of `dmlChannels`, this constant looks like no effect